classes/Task/Restore/RestoreFiles.php

```diff
@@ -48,7 +48,7 @@ class RestoreFiles extends AbstractTask
      */
     public function run(): int
     {
-        $this->container->getState()->setProgressPercentage(
```

Comment:
same for variable `$state`

Reply:
Done, even though "Outdated" does not appear here.